python/src/iceberg/types.py

```diff
@@ -206,19 +204,14 @@ class IntegerType(Type):
 
     min: int = -2147483648
 
-    def __init__(self):
-        super().__init__("int", "IntegerType()", is_primitive=True)
 
-
-class LongType(Type):
+class LongType(PrimitiveType):
     """A Long data type in Iceberg can be represented using an instance of this class. Longs in Iceberg are
     64-bit signed integers.
-
     Example:
         >>> column_foo = LongType()
         >>> isinstance(column_foo, LongType)
         True
-
```

Comment:
Looks like there are a lot of whitespace changes. We generally avoid these because they can cause git commit conflicts. I think we also use an autoformatter for python.

Reply:
hmm I did run the tox formatter before committing `tox -e format` maybe that case was that it was not run on `types.py` previously? (which I think would have been my fault still...)

Reply:
Now I see I think I added the whitespace in the docstrings. They seem much clearer with a line in between the descriptions and examples. Would you like me to get rid of them?

Reply:
I'd prefer keeping the extra whitespace. Maybe we need to fix tox?

Reply:
I think it's actually a matter of black not formatting docstrings. Seems to be an unresolved issue https://github.com/psf/black/issues/144 mentions some additional formatting libraries but I've never seen them. Added in the whitespace manually for now for all docstrings